update_maker/update_maker.py

```diff
@@ -39,6 +44,22 @@ def __init__(self, board: str):
         self._cache.update(raise_on_error=False)
         self._cache.open()
 
+        # copying the context of the real root apt source.list file into the local one
+        if stat(OLU_APT_SOURCES_FILE).st_size == 0 or not isfile(OLU_APT_SOURCES_FILE):
+            copyfile(SYSTEM_APT_SOURCES_FILE, OLU_APT_SOURCES_FILE)    
+        
+            # adding OreSat Debian apt repo
+            with open(OLU_APT_SOURCES_FILE, 'a') as f:
+                f.write('deb [trusted=yes] https://debian.oresat.org/packages ./') 
+                f.close() 
```

Comment:
`f.close()` is redundant, the `with open()` will automatically handle closing the file.

Reply:
I removed `f.close()` and rebased the commits